docs/resources/widgets/Concepts/composing-widgets.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Composing Widgets
+sidebar_position: 2
+---
+
+# Composing widgets
```

Comment:
I feel like we're repeating a lot of the information from the "widgets.md" file here. I don't think we need to list row, column, etc in both files. I would just keep that information here?